gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscoveryMessages.java

```diff
@@ -275,4 +275,7 @@ void roleConfigurationPropertyHasChanged(String propertyName,
 
   @Message(level = MessageLevel.DEBUG, text = "Found downscale event for role: {0} hosts: {1}")
   void foundDownScaleEvent(String role, Set<String> hosts);
+
+  @Message(level = MessageLevel.DEBUG, text = "Service discovery client connect timeout {0}ms, read timeout {1}ms, write timeout {2}ms")
```

Comment:
I would set this one on `INFO` level and add a space before all `ms` occurrences in the above `text` field.

Reply:
The `getClient()` is called before every service discovery. I can change it to `INFO` but then probably we should put the logging elsewhere to ensure that it's not called too frequently.

Reply:
ACK.